Makefile

```diff
@@ -31,6 +32,12 @@ run-camera-simulator:
 run-telegraf:
 	cd telegraf && $(MAKE) run
 
+run-portainer:
+	docker compose -p portainer -f docker-compose-portainer.yml up -d
```

Comment:
since we are using docker compose v2, we should change the prerequisites list to include Docker compose v2 in documentation: https://github.com/intel-retail/automated-self-checkout#prerequisites